core/templates/pages/exploration-editor-page/services/exploration-rights.service.spec.ts

```diff
@@ -301,8 +317,8 @@ describe('Exploration rights service', function() {
     expect(ers.checkUserAlreadyHasRoles('notInAllUsersList')).toBeFalsy();
   });
 
-  it('should check oldrole of user', function() {
-    var sampleDataResultsCopy = angular.copy(sampleDataResults);
+  it('should check oldrole of user', () => {
+    let sampleDataResultsCopy = angular.copy(serviceData);
```

Comment:
Try not to use anything from the global angular object while migrating a file. The global angular object won't be available once angularjs library is removed from the page. Use `cloneDeep` from lodash instead.

Reply:
Done